src/i965_drv_video.c

```diff
@@ -1044,7 +1044,7 @@ i965_GetConfigAttributes(VADriverContextP ctx,
                     profile == VAProfileH264StereoHigh ||
                     profile == VAProfileH264MultiviewHigh) {
                     if (IS_GEN9(i965->intel.device_info))
-                        attrib_list[i].value = (2 << 16) | (8 << 0);
+                        attrib_list[i].value = (1 << 16) | (4 << 0);
```

Comment:
Does AVC encoder support 4 fwd and 1 bwd ref frames for BDW/BSW? BDW/BSW also use the new shaders now

Reply:
@QuPengfei could you update the patch for BDW/BSW too?

Reply:
add support for BDW/BSW. thanks